datadog-logs-oci-orm/logs-function/README.md

```diff
@@ -0,0 +1,41 @@
+# Service Connector Hub function
+OCI function used for log forwarding. See [DataDog documentation for more info](https://docs.datadoghq.com/integrations/oracle_cloud_infrastructure/?tab=serviceconnectorhub#oci-function).
+
+## Send Test Log Events to Datadog
+This guide will help you test sending log events to Datadog. Follow the steps below to install dependencies, configure your API key, and send test log events.
+
+### Step 1: Install Dependencies
+First, install the required dependencies listed in the `requirements.txt` file. You can do this using `pip`:
+
+```sh
+pip install -r requirements.txt
```

Comment:
Not sure if this should be done as a part of this PR but IMO it is not clear to customers where they should run this. eg cloud shell? IMO [the docs we currently have](https://docs.datadoghq.com/integrations/oracle_cloud_infrastructure/?tab=serviceconnectorhub#oci-function) could use more hand-holding, for example how does one create a boilerplate python function.

Reply:
For the purpose of this project, this readme can be considered applicable only for developers who are making changes to the python function. That includes mostly the Datadog team, but if there are external contributors, they can benefit from these steps as well. These steps shouldn't be executed from within customer's OCI environment.